test/ios/MapViewTests.m

```diff
@@ -220,6 +220,26 @@ - (void)testPan {
                       @"panning map right should decrease center longitude");
 }
 
+- (void)testSetCenterCancelsTransitions {
+    CLLocationCoordinate2D dc = CLLocationCoordinate2DMake(38.894368, -77.036487);
+    CLLocationCoordinate2D dc_west = CLLocationCoordinate2DMake(38.894368, -77.076487);
+    [tester.mapView setCenterCoordinate:dc animated:NO];
+    [tester.mapView setCenterCoordinate:dc_west animated:YES];
+    dispatch_after(dispatch_time(DISPATCH_TIME_NOW, 0.15 * NSEC_PER_SEC),
```

Comment:
I think this is actually problematic because KIF/XCTest won't necessarily wait for this async dispatch before assuming the test passed. We should perhaps `dispatch_after()`, then follow up with a `[tester waitForAnimationsToFinish];`, then test that the conditions inside the dispatch were met. For example, we should set a variable there, then after waiting, test that the variable is as expected. Otherwise there is no way to guarantee that the block is run. 

More: http://www.pumpmybicep.com/2014/08/20/asynchronous-unit-testing-with-xctest/